server/chat/backend/agent/utils/persistence/context_manager.py

```
@@ -239,7 +240,7 @@ def _serialize_messages(self, processed_messages):
         return serialized
 
     @staticmethod
-    def _upsert_session(cursor, conn, session_id, user_id, context_json, now):
+    def _upsert_session(cursor, conn, session_id, user_id, org_id, context_json, now):
```

Ruff (0.15.12)

[warning] Missing return type annotation for staticmethod `_upsert_session`

Add return type annotation: `bool | None`

(ANN205)